tests/apollo/test_skvbc_dbsnapshot.py

```diff
@@ -931,7 +931,7 @@ async def test_restore_from_snapshot_of_other(self, bft_network, tracker):
 
         crashed_replica = list(bft_network.random_set_of_replicas(1, {initial_prim}))[0]
         bft_network.stop_replica(crashed_replica)
-        await skvbc.send_n_kvs_sequentially(DB_CHECKPOINT_WIN_SIZE)  # run till the next checkpoint
+        await skvbc.send_n_kvs_sequentially(DB_CHECKPOINT_WIN_SIZE + 100)  # run till the next checkpoint
```

Comment:
Do you want to pass the next checkpoint?

Reply:
yes, in order to test that we don't have state deviation (as we had before this fix)

Reply:
Maybe multiply instead of adding a constant for the day when we make DB_CHECKPOINT_WIN_SIZE configurable.